typings/codePush.d.ts

```diff
@@ -7,7 +7,15 @@
 
 declare module Http {
     export const enum Verb {
-        GET, HEAD, POST, PUT, DELETE, TRACE, OPTIONS, CONNECT, PATCH
+        GET = "GET",
+        HEAD = "HEAD",
+        POST = "POST",
+        PUT = "PUT",
+        DELETE = "DELETE",
+        TRACE = "TRACE",
+        OPTIONS = "OPTIONS",
+        CONNECT = "CONNECT",
+        PATCH = "PATCH",
```

Comment:
Is there a typescript version limitation that would prevent string enums from used in this project? I can remove this before merge if that's the case.

Reply:
@alexandergoncharov  I guess since 1bee4b1b73ec47b0429e24a99a56810d290c80ab is a thing, this should be fine.